processors/src/VertexAnaProcessor.cxx

```diff
@@ -283,6 +308,10 @@ bool VertexAnaProcessor::process(IEvent* ievent) {
                 pos_trk,
                 weight);
 
+        _vtx_histos->Fill1DHisto("vtx_Psum_h", p_ele.P()+p_pos.P(), weight);
+        _vtx_histos->Fill1DHisto("vtx_Esum_h", ele_E + pos_E, weight);
```

Comment:
It seems to me the sum of the momenta/energy of the ele and pos are vtx level plots, and not track level plots. That is why I did this, because I didn't see them at the vtx level, and didn't look at the track level for them. Doesn't look like I even instantiate these guys in vtxSelection_2019 atm. This does seem like the correct place for them to me though.

Reply:
Hm. The whole plotting support for Vtx I think is becoming messy and we probably should separate plots for tracks from plots from Vertices. I'd prefer to keep all the plotting in one place and not have parts in the processor and parts in the histo class. 
But fine for now.  I'll change this